packages/react/src/core/tokens.ts

```diff
@@ -119,6 +119,17 @@ const transition = {
 	timingFunction: 'ease', // https://developer.mozilla.org/en-US/docs/Web/CSS/transition-timing-function
 };
 
+const zIndex = {
+	base: 0,
+	elevated: 1,
+	overlay: 100,
+	dialog: 110,
+	popover: 200,
+	skipLink: 999,
+};
+
+export type zIndex = keyof typeof zIndex;
```

Comment:
Should this be a capital to make it different to the object name?

```suggestion
export type ZIndex = keyof typeof zIndex;
```